docs/src/test-components-js.md

```diff
@@ -853,6 +853,14 @@ export default defineConfig({
 });
 ```
 
+### I have a component that imports CSS. How can I configure Vite to handle CSS files?
+
+If you have a component that imports a CSS file, Vite will handle it automatically. You can also use CSS Pre-processors such as Sass, Less, or Stylus and Vite will handle them as well without any additional configuration. However, corresponding CSS Pre-processor needs to be installed.
```

Comment:
```suggestion
If you have a component that imports CSS, Vite will handle it automatically. You can also use CSS pre-processors such as Sass, Less, or Stylus, and Vite will handle them as well without any additional configuration. However, corresponding CSS pre-processor needs to be installed.
```

Reply:
I like to have it lowercase as you suggest it, but Vite have in uppercase in the docs. But changed as you wish.